e2e/utils/index.ts

```diff
@@ -803,7 +803,8 @@ export function expectTestsPass(v: { stdout: string; stderr: string }) {
 
 export function runCommand(
   command: string,
-  options?: Partial<ExecSyncOptions>
+  options?: Partial<ExecSyncOptions>,
+  failOnErr = false
 ): string {
```

Comment:
non-blocking: can we make this an options object instead? do we don't have to provide other args if we don't want/need to?

```
export function runCommand(command: string, options?: {exec?: Partial<ExecSyncOptions>, failOnErr: boolean} = {failOnError = false}
```

Reply:
Done